### PR TITLE
[zh-cn]: Object proofread part 1

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

校对并修正一些小的错漏。按照 MDN 目录索引，本部份到 defineProperty()

### Motivation

同上

### Additional details

🈚️

### Related issues and pull requests

🈚️

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/object/__definegetter__/index.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ __defineGetter__(prop, func)
...
 
-`__defineGetter__()` 在规范中被定义为“规范性可选”，这意味着不需要任何实现来实现它。然而，所有主要的浏览器都实现了它，并且由于它的持续使用，它不太可能被删除。如果一个浏览器实现了 `__defineGetter__()`，那么它还需要实现 [`__lookupGetter__()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/__lookupGetter__)、[`__lookupSetter__()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/__lookupSetter__) 和 [`__defineSetter__()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineSetter__) 方法。
+`__defineGetter__()` 在规范中被定义为“规范性可选项”，这意味着不需要任何实现来实现它。然而，所有主要的浏览器都实现了它，并且由于它的持续使用，它不太可能被删除。如果一个浏览器实现了 `__defineGetter__()`，那么它还需要实现 [`__lookupGetter__()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/__lookupGetter__)、[`__lookupSetter__()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/__lookupSetter__) 和 [`__defineSetter__()`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineSetter__) 方法。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/__definegetter__/index.md:38]: "Normative" is an adjective, so way not "规范可选项" or "规范可选"?

Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/object/__definegetter__/index.md:38]: "Normative optional" means that the standard's implementation was not required but if there is any implementation, it must be followed. Even the "normative" is an adjective here, I prefer to exchange the part of speech of two words. "可选实现的规范" or "实现可选的规范" are alternatives in my options.

Comment (yin1999) [reply files/zh-cn/web/javascript/reference/global_objects/object/__definegetter__/index.md:38]: I found "引用标准" in <https://zh.wikipedia.org/wiki/ISO/IEC_20000>, so I'm thinking about coud we use "可选标准", as the translation could be found in <https://www.chinaw3c.org/category/wai-domain/7/>, and this should be opposite of "强制标准".

Addition: I cound found the term - "规范性引用文件" in <http://www.china-cas.org/u/cms/www/201612/021120208ygk.pdf>.

Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/object/__definegetter__/index.md:38]: I think "可选标准" is a good choice.

File: files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/__lookupGetter__
 
 {{JSRef}} {{deprecated_header}}
 
-> **备注：** 该特性已被弃用，建议使用 {{jsxref("Object.getOwnPropertyDescriptor()")}} API。该方法的行为只是出于 web 兼容性进行了规定，在任何平台上都不要求实现该方法。它可能无法在所有地方正常工作。
+> **备注：** 该方法已被弃用，建议使用 {{jsxref("Object.getOwnPropertyDescriptor()")}} API。该方法的行为只针对 Web 兼容性进行了规定，在任何平台上都不需要实现该方法。它可能无法在所有地方正常工作。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md:8]: 英文文档中使用的都是“feature” 同 deprecated_header 宏的说明保持一致即可

image

File: files/zh-cn/web/javascript/reference/global_objects/object/__definesetter__/index.md [hunk 3]
```diff
@@ -68,7 +68,7 @@ console.log(o.value); // undefined
 console.log(o.anotherValue); // 5
 ```
 
-你也可以使用 {{jsxref("Object.defineProperty()")}} 在对象创建后为其定义一个 setter。与 `__defineSetter__()` 相比，此方法允许你控制 setter 的可枚举性和可配置性，以及定义 [symbol](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol) 属性。`Object.defineProperty()` 方法也适用于 [`null` 原型对象](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object#null_原型对象)，因为它们没有继承 `Object.prototype`，所以没有 `__defineSetter__()` 方法。
+你也可以使用 {{jsxref("Object.defineProperty()")}} 在对象创建后为其定义一个 setter。与 `__defineSetter__()` 相比，此方法允许你控制 setter 的可枚举性和可配置性，以及定义 [Symbol](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol) 属性。`Object.defineProperty()` 方法也适用于 [`null` 原型对象](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object#null_原型对象)，因为它们没有继承 `Object.prototype`，所以没有 `__defineSetter__()` 方法。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/__definesetter__/index.md:71]: 用小写没关系，不用特地改

image

Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/__definesetter__/index.md:71]:
```suggestion
你也可以使用 {{jsxref("Object.defineProperty()")}} 在对象创建后为其定义一个 setter。与 `__defineSetter__()` 相比，此方法允许你控制 setter 的可枚举性和可配置性，以及定义 [symbol](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Symbol) 属性。`Object.defineProperty()` 方法也适用于 [`null` 原型对象](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object#null_原型对象)，因为它们没有继承 `Object.prototype`，所以没有 `__defineSetter__()` 方法。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md (hunk 2 above)
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/__lookupgetter__/index.md:8]:
```suggestion
> **备注：** 该特性已被弃用，建议使用 {{jsxref("Object.getOwnPropertyDescriptor()")}} API。该方法的行为只针对 Web 兼容性进行了规定，在任何平台上都不需要实现该方法。它可能无法在所有地方正常工作。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/__definegetter__/index.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/__defineGetter__
 
 {{JSRef}}{{Deprecated_Header}}
 
-**注意：** 此方法已弃用，建议使用[对象初始化语法](/zh-CN/docs/Web/JavaScript/Reference/Operators/Object_initializer)或 {{jsxref("Object.defineProperty()")}} API 来定义 [getter](/zh-CN/docs/Web/JavaScript/Reference/Functions/get)。该方法的行为只针对 Web 兼容性进行了规定，在任何平台上都不需要实现该方法。它可能无法在所有地方正常工作。
+> **备注：** 此方法已弃用，建议使用[对象初始化语法](/zh-CN/docs/Web/JavaScript/Reference/Operators/Object_initializer)或 {{jsxref("Object.defineProperty()")}} API 来定义 [getter](/zh-CN/docs/Web/JavaScript/Reference/Functions/get)。该方法的行为只针对 Web 兼容性进行了规定，在任何平台上都不需要实现该方法。它可能无法在所有地方正常工作。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/__definegetter__/index.md:8]: 同

File: files/zh-cn/web/javascript/reference/global_objects/object/__definesetter__/index.md [hunk 5]
```diff
@@ -5,9 +5,9 @@ slug: Web/JavaScript/Reference/Global_Objects/Object/__defineSetter__
 
 {{JSRef}}{{Deprecated_Header}}
 
-> **备注：** 此方法已弃用，建议使用[对象初始化语法](/zh-CN/docs/Web/JavaScript/Reference/Operators/Object_initializer)或 {{jsxref("Object.defineProperty()")}} API 来定义 [setter](/zh-CN/docs/Web/JavaScript/Reference/Functions/set)。该方法的行为只是出于 Web 兼容性进行了规定，在任何平台上都不要求实现该方法。它可能无法在所有地方正常工作。
+> **备注：** 此方法已弃用，建议使用[对象初始化语法](/zh-CN/docs/Web/JavaScript/Reference/Operators/Object_initializer)或 {{jsxref("Object.defineProperty()")}} API 来定义 [setter](/zh-CN/docs/Web/JavaScript/Reference/Functions/set)。该方法的行为只针对 Web 兼容性进行了规定，在任何平台上都不需要实现该方法。它可能无法在所有地方正常工作。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/__definesetter__/index.md:8]: 同

File: files/zh-cn/web/javascript/reference/global_objects/object/__definegetter__/index.md (hunk 4 above)
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/__definegetter__/index.md:8]:
```suggestion
> **备注：** 此特性已弃用，建议使用[对象初始化语法](/zh-CN/docs/Web/JavaScript/Reference/Operators/Object_initializer)或 {{jsxref("Object.defineProperty()")}} API 来定义 [getter](/zh-CN/docs/Web/JavaScript/Reference/Functions/get)。该方法的行为只针对 Web 兼容性进行了规定，在任何平台上都不需要实现该方法。它可能无法在所有地方正常工作。
```

File: files/zh-cn/web/javascript/reference/global_objects/object/__definesetter__/index.md (hunk 5 above)
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/object/__definesetter__/index.md:8]:
```suggestion
> **备注：** 此特性已弃用，建议使用[对象初始化语法](/zh-CN/docs/Web/JavaScript/Reference/Operators/Object_initializer)或 {{jsxref("Object.defineProperty()")}} API 来定义 [setter](/zh-CN/docs/Web/JavaScript/Reference/Functions/set)。该方法的行为只针对 Web 兼容性进行了规定，在任何平台上都不需要实现该方法。它可能无法在所有地方正常工作。
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**